jest/jest.config.js

```diff
@@ -28,6 +28,15 @@ module.exports = {
   ...(process.env.CI
     ? {
         coverageReporters: ['json', 'lcov', 'clover'],
+        reporters: [
+          'default',
+          [
+            'jest-junit',
+            {
+              outputName: 'jest.junit.xml',
+            },
```

Comment:
this will make it so you can have nicely formatted test names in the codecov UI and comment [[codecov docs](https://docs.codecov.com/docs/test-analytics#1-output-a-junit-xml-file-in-your-ci)] [[jest-junit docs](https://github.com/jest-community/jest-junit?tab=readme-ov-file#configuration)]

```suggestion
            {
              outputName: 'jest.junit.xml',
              classNameTemplate: '{filepath}',
            },
```